source/s3_auto_ranged_put.c

```diff
@@ -480,7 +462,9 @@ static bool s_s3_auto_ranged_put_update(
                 request = aws_s3_request_new(
                     meta_request,
                     AWS_S3_AUTO_RANGED_PUT_REQUEST_TAG_LIST_PARTS,
```

Comment:
Not exactly for this PR, but do we still need the separated tags for each meta request type? Feel like it's a repeated info here.

Reply:
There's still 1 case where the meta-request has 2 tags that use the same-request type:
https://github.com/awslabs/aws-c-s3/blob/f961971f3851782d40126f0597f007c73ba4ae0c/source/s3_auto_ranged_get.c#L49-L51

🤷‍♀️

Reply:
We talked, and we'd all like to remove the concept of "request tag".
But that's a sizable change, so we're going to tackle it in the future.
I added a TODO